modules/trusted-firmware-m/Kconfig

```diff
@@ -233,6 +233,42 @@ config TFM_REGRESSION
 	  When enabled, this option signifies that the TF-M build includes
 	  the Secure and the Non-Secure regression tests.
 
+choice TFM_PSA_TEST
+	prompt "Enable a PSA test suite"
+	default TFM_PSA_TEST_NONE
```

Comment:
Question: are we fine with NONE, or we _always_ would need one suite to be selected?

Reply:
This is instead of a bool switch for enabling/disabling PSA tests. The alternative is to enclose the choice in an `if`.